doc/develop/extending.rst

```diff
@@ -188,23 +194,42 @@ Open the *Anaconda prompt* and type::
    pip install git+https://github.com/delmic/Pyro4.git
 
 Download and install `Microsoft Visual C++ Compiler for Python 2.7 <https://www.microsoft.com/download/details.aspx?id=44266>`_.
-If you use Python 3.6, download and install `Build Tools for Visual Studio 2019 <https://www.visualstudio.com/downloads/#build-tools-for-visual-studio-2019>`_. 
 
+Python 3 setup
+""""""""""""""
+For Python 3, open the *Anaconda prompt* and type::
+
+   cd Documents\odemis
+   conda create -y --name odemisdev python==3.6.8
+   conda activate odemisdev
+   conda config --append channels conda-forge
+   conda install --name odemisdev --file requirements-py3.txt
+   conda develop src
+   pip install git+https://github.com/delmic/Pyro4.git
+
```

Comment:
Maybe nice as a task in the back-log wouldn't it be nice to write an automatic .exe/installer which does all of this for the user?

Reply:
Even better would be to have Odemis packaged in the anaconda repo (or pypi)... I've just added the task to the backlog.